### DISCUSSION TITLE
How to use dashboard-ReadLineLength.css in editing mode

### DISCUSSION THREAD

**Discussion by albertoloscerritos** [TfTHacker/DashboardPlusPlus#18] [Q&A] [answered]
Hi, does anybody knows if it is possible to enable the wide mode using a snippet, but make the wide view visible also in editing mode. Since the dashboard-ReadLineLength.css snippet only makes it visible in reading mode. 
Thanks.

**Comment by LordMartron94** [comment, marked as answer]
I made it a big view class so I can use it for normal notes too:
/* CSS Snippet for Obsidian */
.bigview .markdown-preview-section {
    max-width: 100% !important;
    width: 100% !important;
}

.bigview {
    --file-line-width: 100%;
}

**Reply by albertoloscerritos** [reply]
Sir, you saved my day.
Thanks :)

**Reply by albertoloscerritos** [reply]
I just tried it for a bit. And I don't know if this is a bug or is just me, but when I use the snippet in one note, while I am in editing mode, and toggle between notes (notes that don't have the snippet enables), it affects some of the other notes i.e. some of the other notes become wide. If I keep toggling between notes, some get back to normal and some do not. Have this happened to you? I disabled all other snippets I am running 1.4.6.
Thanks
Thanks
